src/ex4/server/routes/v1/task.routes.js

```diff
@@ -0,0 +1,21 @@
+import express from "express";
+import {
+    readAndPrintAllTodos,
+    handleInput,
+    deleteTodo,
+} from "../../services/item_manager.js";
+//this route file import controller stuff
+const router = express.Router();
+
+router
```

Comment:
<img width="355" alt="image" src="https://user-images.githubusercontent.com/73115367/174613189-0be04a17-8b84-440c-af90-30a8f22be83b.png">
I would recommend using this syntax, as it more readable and allow you to handle the response if you need to